int16 dof1
int16 dof2
int16 dof3
int16 dof4
int16 dof5
int16 dof6
